apps/condo/domains/ticket/schema/Ticket.js

```diff
@@ -333,6 +334,26 @@ const Ticket = new GQLListSchema('Ticket', {
                 resolvedData.propertyAddress = property.address
                 resolvedData.propertyAddressMeta = property.addressMeta
             }
+            if (!resolvedData.contact) {
```

Comment:
??? we don't need to create a contact automatically ???

Reply:
If contact data was sended, I don't think we should overwrite it